### PR TITLE
Update to current reality

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Can't blame a guy for trying.

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/podman.io/issues/391

File: whatis.md [hunk 1]
```diff
@@ -29,19 +29,20 @@ At a high level, the scope of libpod and Podman is the following:
 
 ### Roadmap
 
-*   Allow the Podman CLI to use a REST API to connect to remote Podman services on Linux from Mac and Windows.
-*   Integrate libpod into CRI-O to replace its existing container management backend.
 *   Further work on the podman pod command.
 *   Further improvements on rootless containers.
+*   Support for rootless Podman with NFS Homedirs
```
Comment (TomSweeneyRedHat) [inline whatis.md]:
```suggestion
*   Support for rootless Podman with NFS Homedirs.
```

File: whatis.md [hunk 2]
```diff
@@ -29,19 +29,20 @@ At a high level, the scope of libpod and Podman is the following:
...
+*   Support for rootless Podman on native overlayfs
```
Comment (TomSweeneyRedHat) [inline whatis.md]:
```suggestion
*   Support for rootless Podman on native overlayfs.
```

File: whatis.md [hunk 3]
```diff
@@ -29,19 +29,20 @@ At a high level, the scope of libpod and Podman is the following:
...
+*   Enhancements to speedup pulling of images
```
Comment (TomSweeneyRedHat) [inline whatis.md]:
```suggestion
*   Enhancements to speedup pulling of images.
```

File: whatis.md [hunk 4]
```diff
@@ -29,19 +29,20 @@ At a high level, the scope of libpod and Podman is the following:
...
+*   Podman machine for setting up VMs for running podman -remote run nativley on Mac and Linux
```
Comment (TomSweeneyRedHat) [inline whatis.md]:
```suggestion
*   Podman machine for setting up VMs for running `podman -remote run` nativley on Mac and Linux.
```

File: whatis.md [hunk 5]
```diff
@@ -10,11 +10,11 @@ title: What is podman?
...
 
-The Podman service runs only on Linux platforms, however a REST API and clients are currently under development which will allow Mac and Windows platforms to call the service. There is currently a Varlink based [remote client](https://github.com/containers/podman/blob/master/docs/tutorials/remote_client.md) which runs on Mac or Windows platforms that allows the remote client to talk to the Podman server on a Linux platform. In addition to those clients, there is also a [Mac client](https://github.com/containers/podman/blob/master/docs/tutorials/mac_client.md). **NOTE:** the Varlink remote client will be deprecated after the REST API is completed.
+The Podman service runs only on Linux platforms, however the podman remote REST API client exists which on Mac and Windows platforms and can communicate with the Podman service running on a Linux machine or VM via ssh. [Mac client](https://github.com/containers/podman/blob/master/docs/tutorials/mac_client.md).
```
Comment (TomSweeneyRedHat) [inline whatis.md]:
```suggestion
The Podman service runs only on Linux platforms, however the podman remote REST API client exists on Mac and Windows platforms and can communicate with the Podman service running on a Linux machine or VM via ssh. [Mac client](https://github.com/containers/podman/blob/master/docs/tutorials/mac_client.md).
```

**Comment by TomSweeneyRedHat** [comment]
A few minor nits, otherwise LGTM
@mheon WDYT?

**Comment by TomSweeneyRedHat** [comment]
LGTM
@mheon could you take a quick peruse please?

File: whatis.md [hunk 6]
```diff
@@ -29,19 +29,20 @@ At a high level, the scope of libpod and Podman is the following:
...
 ### Roadmap
 
-*   Allow the Podman CLI to use a REST API to connect to remote Podman services on Linux from Mac and Windows.
-*   Integrate libpod into CRI-O to replace its existing container management backend.
 *   Further work on the podman pod command.
 *   Further improvements on rootless containers.
+*   Support for rootless Podman with NFS Homedirs.
+*   Support for rootless Podman on native overlayfs.
+*   Enhancements to speedup pulling of images.
+*   Podman machine for setting up VMs for running `podman -remote run` nativley on Mac and Linux.
```
Comment (mheon) [inline whatis.md:37]: I'd just rephrase this to "Improved support for running on OS X"

**Comment by mheon** [comment]
One nit otherwise LGTM

**Comment by TomSweeneyRedHat** [comment]
@baude, merging now, please address @mheon's concern
